indra/newview/llvoavatarself.cpp

```diff
@@ -1288,6 +1288,50 @@ U32 LLVOAvatarSelf::getNumWearables(LLAvatarAppearanceDefines::ETextureIndex i)
 	return gAgentWearables.getWearableCount(type);
 }
 
+void LLVOAvatarSelf::postMotionUpdate() 
+{
```

Comment:
The viewer is sending every animation frame... in crowded regions this could add up. We could send every Nth frame instead.

Reply:
Perhaps an expiry logic flow would be more tunable.  Add `mNextPostMotionUpdateExpiry`, set it to a future time, and only send an update when expired.  The period between each expiry could be adjusted at runtime if necessary.  Note, the server would be the most susceptible to overload in a crowded situation and would need protection.  It should: enforce its own dynamic version of the update rate it will accept and discard client updates that exceed it.  Bonus points if the server could tell the viewer the update rate it is enforcing on its end.